lib/kits/core-ui/list-publications.js

```diff
@@ -112,8 +112,8 @@ const ListPublications = (
         });
 
     const dispatchPublicationClickEvent = (detail) => {
-        window.dispatchEvent(
-            new CustomEvent('tjek-publication-list-item-clicked', {
+        document.querySelector(scriptEls.mainContainer)?.dispatchEvent(
+            new CustomEvent('publication_list:clicked', {
```

Comment:
Maybe publication:clicked - does it matter it's a list?

Reply:
Ah yeah, good point. It doesn't matter if it's a list since it's the publication that's clicked. Will rename this :)